app/src/components/BaseHeader.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <header class="px-4 md:px-12 py-20 border-b border-grey-100 text-center relative">
+    <!-- optional breadcrumb within a header -->
+    <Breadcrumb v-if="breadcrumbContent" :path="breadcrumbContent" />
+
+    <h1>{{ name }}</h1>
+
+    <!-- optional subline -->
+    <h1 v-if="tagline">“{{ tagline }}”</h1>
+
+    <!--optional by address last seen / last  updated " "byWhen"  -->
+    <div v-if="by">
+      <span class="text-grey-400 mr-4">by</span>
+      <a class="link mr-4" :href="`https://etherscan.io/address/${owner}`">{{ by }}</a>
+      <span v-if="byWhen" class="italic text-grey-400">{{ byWhen }}</span>
+    </div>
+
+    <!-- optional quote -->
+    <p v-if="quote" class="intent italic mt-8 mx-auto max-w-6xl">“{{ quote }}”</p>
+
+    <span>
+      <!-- optional navigation to jump to next item in a list -->
+      <router-link
+        :class="['group', 'inline', 'p-6', 'absolute', 'right-0', !nextPath ? 'cursor-default' : '']"
+        style="top: 50%; transform: translateY(-50%)"
+        :to="nextPath"
+      >
```

Comment:
Should these router links have `v-if="nextPath"` (and same for last path) on them? When we use this component on e.g. the Cart page, we don't want the arrows to show up, but it seems they still will be there. Downside is then you can't get the "visible but disable" look we currently use without an extra prop or a special string for `nextPath`

Reply:
Looks like the arrows don't show up, and we just get an empty `span` tag, which is ok but would be a little cleaner to use a `v-if` I think so we don't have unused HTML